packages/util/src/logging.ts

```diff
@@ -58,13 +83,7 @@ export class Logging {
         if (typeof msg !== "string") {
             msg = this.stringify(msg);
         }
-        if (isNode) {
-            // tslint:disable-next-line:no-console
-            console.log(`[${n}] ${Level[level].toUpperCase()} ${id}:  ${msg}`);
-        } else {
-            // tslint:disable-next-line:no-console
-            console.log(`[${n}] %c${Level[level].toUpperCase()}%c ${id}:  ${msg}`, `color:${colours[Level[level]]}`, "");
-        }
+        this._writer.write(n, level, id, msg);
```

Comment:
Minor: _n_ is a little cloudy in this context. perhaps a more descriptive param?